.github/workflows/build_wheels.yml

```diff
@@ -321,8 +332,12 @@ jobs:
     if: needs.check_env_variables.outputs.gcp-variables-set == 'true' && github.event_name != 'pull_request'
     strategy:
       matrix:
-        os : [ubuntu-latest, macos-latest, windows-latest]
+        os : [[self-hosted, ubuntu-20.04], macos-latest, [self-hosted, windows-server-2019]]
```

Comment:
Do we use self-hosted windows here? If so then other places of `windows-latest` may also use self-hosted

Reply:
It was intended to be "windows-latest", my apologies, there are some issues with newer versions of the self-hosted runner agent and these workflows, we're moving on with self-hosted for ubuntu-20.04 for now.